xds/internal/balancer/outlierdetection/e2e_test/outlierdetection_test.go

```diff
@@ -159,9 +159,9 @@ func (s) TestOutlierDetectionAlgorithmsE2E(t *testing.T) {
   "loadBalancingConfig": [
     {
       "outlier_detection_experimental": {
-        "interval": 50000000,
-		"baseEjectionTime": 100000000,
```

Comment:
Can you normalize these in case we decide to be more restrictive in Duration parsing one day, we won't need to come back and change these?

E.g. `"0.050s"` - 3/6/9 digits after decimal & don't omit leading digit.

Reply:
Done.